Sources/Scheduler.swift

```diff
@@ -264,16 +264,16 @@ public final class QueueScheduler: DateSchedulerProtocol {
 		precondition(interval >= 0)
 		precondition(leeway >= 0)
 
-		let nsecInterval = interval * Double(NSEC_PER_SEC)
-		let nsecLeeway = leeway * Double(NSEC_PER_SEC)
+		let msecInterval = interval * 1000
+		let msecLeeway = leeway * 1000
 
 		let timer = DispatchSource.makeTimerSource(
 			flags: DispatchSource.TimerFlags(rawValue: UInt(0)),
 			queue: queue
 		)
 		timer.scheduleRepeating(wallDeadline: wallTime(with: date),
-		                        interval: .nanoseconds(Int(nsecInterval)),
-		                        leeway: .nanoseconds(Int(nsecLeeway)))
+		                        interval: .milliseconds(Int(msecInterval)),
```

Comment:
I love the idea, I started working through what that would take, and we have a few options. If we deprecate this function, there are A LOT of places that will need further deprecations and significant changes as we roll out the newer API including ```DispatchTimeInterval``` instead of ```TimeInterval```
How do you feel about keeping this change to milliseconds, deprecating this function, and then creating an issue to adjust all the other functions that rely on this now deprecated API?